corehq/apps/hqwebapp/signals.py

```diff
@@ -52,4 +53,10 @@ def add_failed_attempt(sender, credentials, token_failure=False, **kwargs):
         user.attempt_date = date.today()
 
     user.login_attempts += 1
-    user.save()
+
+    try:
+        user.save()
+    except ResourceConflict:
+        # swallow this exception due ensure the user still sees an auth failed
```

Comment:
*to ensure?

Reply:
91aba1dc7ea493ab46673650be6c470f291854e9